tests/integration/test_async_insert_adaptive_busy_timeout/test.py

```diff
@@ -367,6 +367,6 @@ def test_change_queries_frequency():
     select_log_query = "SELECT timeout_milliseconds FROM system.asynchronous_insert_log ORDER BY event_time DESC LIMIT 50"
     res = node.query(select_log_query)
     for line in res.splitlines():
-        assert int(line) == min_ms
+        assert int(line) <= min_ms
```

Comment:
It seems this change is Ok, because there can be spurious flushes... And the timeout can be actually 0 for some entries.